bgpd/bgp_bmp.c

```diff
@@ -1783,6 +1816,23 @@ static void bmp_active_connect(struct bmp_active *ba)
 		}
 
 		set_nonblocking(ba->socket);
+
+		if (!sockunion_is_null(&ba->addrsrc)) {
+			res_bind = sockunion_bind(ba->socket, &ba->addrsrc, 0,
+						  &ba->addrsrc);
+			if (res_bind < 0) {
+				sockunion2str(&ba->addrsrc, buf, sizeof(buf));
+				zlog_warn(
```

Comment:
zlog_err?

Reply:
The same question still, is this supposed to be an error or warning? `Failed` seems to be related to error?